jenkins/setup-opm-data.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Predefined by environment
+if test -z "$OPM_DATA_ROOT"
+then
+  OPM_DATA_REVISION="master"
+  if grep -q "opm-data=" <<< $ghprbCommentBody
+  then
+    OPM_DATA_REVISION=pull/`echo $ghprbCommentBody | sed -r 's/.*opm-data=([0-9]+).*/\1/g'`/merge
+  fi
+  # Not specified in trigger, use shared copy
+  if [[ "$OPM_DATA_REVISION" = "master" ]] && [[ ! "$OPM_DATA_ROOT_PREDEFINED" == "" ]]
```

Comment:
it would work in bash, but the portable construct is one equals sign. it's the second that is in error, thanks for spotting that.